app/src/main/java/org/mozilla/fenix/library/LibraryPageView.kt

```diff
@@ -30,16 +30,16 @@ open class LibraryPageView(
     protected fun setUiForNormalMode(title: String?) {
         activity?.title = title
         setToolbarColors(
-            ContextCompat.getColor(context, R.color.white_color),
-            context.getColorFromAttr(R.attr.accentHighContrast)
+            context.getColorFromAttr(R.attr.primaryText),
+            context.getColorFromAttr(R.attr.foundation)
```

Comment:
Were these accidentally swapped in your last PR?

Reply:
Yup. I moved them over but the original functions were in the reverse order.